acctz/acctz.proto

```diff
@@ -290,7 +290,11 @@ message RecordResponse {
   // accomplish the request on the system.
   repeated string task_ids = 7;
 
-  // Name of the component where the accounting record originated from
+  // Name of the component in a vendor system configuration where the accounting
+  // record originated from. This helps to indentify the source of the accouting
+  // record when multiple components can generate them.
+  // Eg: component_name = "linecard0"
+  //     component_name = "chassis0/linecard0"
```

Comment:
This really should be the gnmi.Path string representation of the component

Reply:
Updated the description